packages/cli/lib/artifact-generator.js

```diff
@@ -37,33 +38,24 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
 
   /**
    * Checks if current directory is a LoopBack project by checking for
-   * keyword 'loopback' under 'keywords' attribute in package.json.
-   * 'keywords' is an array
+   * .yo-rc.json and the lbVersion is specified.
    */
   checkLoopBackProject() {
     debug('Checking for loopback project');
     if (this.shouldExit()) return false;
-    const pkg = this.fs.readJSON(this.destinationPath('package.json'));
-    const key = 'loopback';
-    if (!pkg) {
-      const err = new Error(
-        'No package.json found in ' +
-          this.destinationRoot() +
-          '. ' +
-          'The command must be run in a LoopBack project.'
-      );
-      this.exit(err);
-      return;
-    }
-    if (!pkg.keywords || !pkg.keywords.includes(key)) {
-      const err = new Error(
-        'No `loopback` keyword found in ' +
-          this.destinationPath('package.json') +
-          '. ' +
-          'The command must be run in a LoopBack project.'
+    let retErr;
+
+    const version = this.config.get('lbVersion');
```

Comment:
The key is from `.yo.rc.json`, can we use a more obvious name such as `loopback.version`?

Reply:
Do we exit if no `lbVersion` is found in the config file?